packages/core/src/utils/setup-package-json.ts

```diff
@@ -224,3 +232,29 @@ const setupPackageJson =
 
 export const validatePackageJson = setupPackageJson(false);
 export const fixPackageJson = setupPackageJson(true);
+
+export const updatePackageJsonExports = async (
+  packageRoot: string,
+  exports: string[],
+) => {
+  const packagePath = path.join(packageRoot, 'package.json');
+  const packageJson: PackageJson = await fse.readJson(packagePath, { fs });
+
+  const packageExports = packageJson.exports! as Record<string, any>;
+
+  const lastKey = Object.keys(packageExports).pop()!;
+  const lastExport = packageExports[lastKey];
+
+  delete packageExports[lastKey];
+  for (const entry of exports) {
+    packageExports[makeRelative(entry)] = makeRelative(entry);
+  }
+  packageExports[lastKey] = lastExport;
```

Comment:
Is it important that the CSS exports go last?

Reply:
They go before the entry for `package.json`, second to last. Having a predetermined order ensures there's no diff between runs.